python/tvm/relay/op/vision/sort.py

```diff
@@ -0,0 +1,31 @@
+"""Argsort operation"""
```

Comment:
I think we shouldn't put sort op under vision since it's not a vision-specific op.